frontend/src/app/components/auth/login/login.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { User } from '../../../model/user';
+import {AuthService} from '../../../services/auth.service';
+
+@Component({
+  selector: 'bm-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+  user: User;
+  loginForm = this.fb.group({
```

Comment:
I would just create a method for Initialisation
```
loginForm: FormGroup;

createLoginForm(){
  this.loginForm = (...)
}

ngOnInit(){
 this.createLoginForm();
}

```

Reply:
same like user property